front_end/src/components/landing_page/search_widget/SearchWidget.js

```diff
@@ -128,20 +125,10 @@ class SearchWidget extends Component {
   }
 }
 
-SearchWidget.propTypes = {
-  // auth: PropTypes.object.isRequired,
-  // logoutUser: PropTypes.func.isRequired,
-};
-
 const mapStateToProps = state => ({
   query: state.query
 });
-// if this.props.query is empty we will not show the Search page
 
-// export default connect(
-//   mapStateToProps,
-//   { saveQuery }
-// )(withStyles(styles)(SearchWidget));
 export default connect(
   mapStateToProps,
   { submitQuery, saveQuery }
```

Comment:
There's a comment about 'connect()' at the bottom of this file that we don't seem to need anymore.

Reply:
got it